lib/src/webview.dart

```diff
@@ -566,3 +613,22 @@ class _WebviewState extends State<Webview> {
     _cursorSubscription?.cancel();
   }
 }
+
+extension WebviewPointerEventKindExt on WebviewPointerEventKind {
```

Comment:
Please remove this extension in favor of translating the values on the C++ side (see https://github.com/jnschulze/flutter-webview-windows/pull/164#discussion_r917651567)

Reply:
Changed. 

Now the `int32_t`->`COREWEBVIEW2_POINTER_EVENT_KIND` mapping happens in `webview_bridge.cc` and it is using `COREWEBVIEW2_POINTER_EVENT_KIND` as a parameter in the  `webview.h`-API instead of an int32.

I hope this is okay, otherwise i can change it back to int and map/cast it in `webview.h`.